target_color: 'red'
agent_color: 'blue'
xmin: -4.0
xmax: 4.0
ymin: -4.0
ymax: 4.0
